sites-enabled/15-chinese.freecodecamp.org.conf

```diff
@@ -30,31 +23,8 @@ server {
     add_header Content-Type text/plain;
   }
 
-  # reverse proxy client (chinese)
-  location / {
-    # app specific configs
-    include snippets/app/learn.org.conf;
-
-    proxy_pass http://client-chn;
-    include snippets/common/proxy-params.conf;
-  }
-
-  # reverse proxy client fonts
-  location ~* \.(woff|ttf)$ {
-    add_header Cache-Control "public, max-age=31536000, immutable";
-
-    proxy_pass http://client-chn;
-    include snippets/common/security.conf;
-    include snippets/common/proxy-params.conf;
-  }
-
-  # reverse proxy api
-  location /api {
-    if (-f /etc/nginx/maintenance/API.txt) {
-      return 503;
-    }
-    proxy_pass http://api;
-    include snippets/common/proxy-params.conf;
+  location ~ ^/(.*)$ {
+    return 302 https://www.freecodecamp.org/chinese/$1;
   }
```

Comment:
I think this is *probably* fine, but it's a different approach to how we do other redirects like
```
# HTTP redirect
server {
  listen 80;
  listen [::]:80;

  server_name .freecodecamp.org;

  return 301 https://www.freecodecamp.org$request_uri;
}
```

Reply:
I think so, yeah.  My understanding was if you have `https://chinese.freecodecamp.org/some-stuff` then `$request_uri` will be `some-stuff`.

The outcome should be the same in both approaches, but using `$request_uri` is more idiomatic.

Not a big deal, tbh, but I thought I'd mention it.